app/views/session_timeout/_warning.html.erb

```diff
@@ -7,10 +7,10 @@
       <hr class='margin-bottom-4 border-width-05' />
       <div class='margin-bottom-6'>
         <p id='<%= description_id %>'>
-          <%= modal.message %>
+          <%= modal.message(self) %>
```

Comment:
I think we could pass in `view_context` to the modal presenter constructor and remove this `self` here:

```suggestion
          <%= modal.message %>
```

https://github.com/18F/identity-idp/blob/9844747bd9c27f5030f5944a3ca09732fe3f1e37/app/helpers/session_timeout_warning_helper.rb#L25-L31

^^ inside a controller it'll be accessible as `view_context` here

Reply:
I like it 👍  Will update.

Reply:
`view_context` wasn't available in the helper, but `self` was. It seems the helper isn't included in a controller, it's automatically available through thew view.